src/core/taylor.rkt

```diff
@@ -121,13 +121,13 @@
              (cons n k)
              (λ ()
                (cond
-                 [(= k 0) (list (build-list n (const 0)))]
+                 [(= k 0) (list (make-list n 0))]
                  [(= n 1) (list (list k))]
                  [(= n 0) '()]
                  [else
-                  (apply append
-                         (for/list ([i (in-range 0 (+ k 1))])
-                           (map (curry cons i) (n-sum-to (- n 1) (- k i)))))]))))
+                  (for*/list ([i (in-range 0 (+ k 1))]
+                              [v (in-list (map (curry cons i) (n-sum-to (- n 1) (- k i))))])
```

Comment:
@jackfirth this is a useful change but it’d be even better to inline map calls into the function body